tests/ui_tests/test_header_and_footer.py

```diff
@@ -31,80 +32,121 @@
+            BasePageObject(app_fs.driver, app_fs.adcm.url).wait_url_contains_path(params["help_link"])
+            close_current_tab(app_fs.driver)
+        header.click_help_button_in_header()
+        with wait_for_new_window(app_fs.driver):
+            header.click_doc_link_in_help_popup()
+        with allure.step(f"Check new opened page: {params['docs_link']}"):
+            BasePageObject(app_fs.driver, app_fs.adcm.url).wait_url_contains_path(params["docs_link"])
+
+    def test_check_header_user_settings_for_authorised_user(self, app_fs):
+        header = PageHeader(app_fs.driver, app_fs.adcm.url)
+        header.click_account_button_in_header()
+        header.check_account_popup()
+        header.click_settings_link_in_acc_popup()
+        header.wait_url_contains_path(AdminSettingsPage(app_fs.driver, app_fs.adcm.url).path)
+
+        header.click_account_button_in_header()
+        header.click_profile_link_in_acc_popup()
+        header.wait_url_contains_path(ProfilePage(app_fs.driver, app_fs.adcm.url).path)
+
+        header.click_account_button_in_header()
+        header.click_logout_in_acc_popup()
+        header.wait_url_contains_path(LoginPage(app_fs.driver, app_fs.adcm.url).path)
+
+    @pytest.mark.xfail(reason="https://arenadata.atlassian.net/browse/ADCM-2054")
+    def test_check_back_button_in_browser_for_header_links(self, app_fs, sdk_client_fs):
+        add_dummy_objects_to_adcm(sdk_client_fs)
+        with allure.step("Check back button for cluster page header link"):
+            intro_page = AdminIntroPage(app_fs.driver, app_fs.adcm.url)
+            intro_page.header.click_cluster_tab_in_header()
+            cluster_page = ClusterListPage(app_fs.driver, app_fs.adcm.url)
+            cluster_page.wait_page_is_opened()
+            cluster_page.driver.back()
+            intro_page.wait_page_is_opened()
+        with allure.step("Check back button for hostprovider page header link"):
+            cluster_page.open()
+            cluster_page.header.click_hostproviders_tab_in_header()
+            hostprovider_page = ProviderListPage(app_fs.driver, app_fs.adcm.url)
+            hostprovider_page.wait_page_is_opened()
+            hostprovider_page.driver.back()
+            cluster_page.wait_page_is_opened()
+        with allure.step("Check back button for hosts page header link"):
+            hostprovider_page.open()
+            hostprovider_page.header.click_hosts_tab_in_header()
+            hosts_page = HostListPage(app_fs.driver, app_fs.adcm.url)
+            hosts_page.wait_page_is_opened()
+            hosts_page.driver.back()
+            hostprovider_page.wait_page_is_opened()
+        with allure.step("Check back button for jobs page header link"):
+            hosts_page.open()
+            hosts_page.header.click_jobs_tab_in_header()
+            jobs_page = JobListPage(app_fs.driver, app_fs.adcm.url)
+            jobs_page.wait_page_is_opened()
+            jobs_page.driver.back()
```

Comment:
Add allure step to this method

Reply:
Probably you can add it as a BasePageObject method

Reply:
added